### PR TITLE
[3.1.0] Add docs to configure IS as extenral IDP using OIDC and SAML

### PR DESCRIPTION
**Pull request by vithu30**
## Purpose
This PR contains following changes:

-  Fixes https://github.com/wso2/docs-apim/issues/922

- Refactored the doc to configure IS as external IDP using SAML SSO in federated scenario

### CODE REVIEW

**Comment by Mariangela** [comment]
@vithu30 , can you please address the review comment regarding the image name and fix the merge conflict. Please note that I have made changes in your fork. So please make sure to pull those changes to your local repo before making any changes.

**Review by Mariangela — CHANGES_REQUESTED** [review]
Please see the comment.

**Comment by vithu30** [comment]
@Mariangela, resolved the conflicts.

File: en/docs/install-and-setup/setup/sso/configuring-identity-server-as-external-idp-using-saml.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+            1.  Click the **SaaS Application** option that appears after registering the service provider.
+
+            [![saas-configuration-in-service-provider]({{base_path}}/assets/img/setup-and-install/saas.png)]({{base_path}}/assets/img/setup-and-install/saas.png)            
+
+            If you do not select the **SaaS Application** option, only users in the current tenant domain will be allowed to login to the portals. You will need to register separate service providers for portals from each tenant.
+
+4.  Upload the public certificate of the API Manager by selecting **Select SP Certificate Type**.
+
+    [![upload-certificate-in-SP-for-saml2-sso.png]({{base_path}}/assets/img/setup-and-install/upload-certificate-in-SP-for-saml2-sso.png)]({{base_path}}/assets/img/setup-and-install/upload-certificate-in-SP-for-saml2-sso.png)
```
[the remark this answers is not in the document]
Comment (Mariangela) [reply en/docs/install-and-setup/setup/sso/configuring-identity-server-as-external-idp-using-saml.md]: did you rename the image itself? I do not see it under the physical files that have been uploaded.

**Review by Mariangela — APPROVED** [review]

**Pull request merged**